static/app/views/app/systemAlerts.tsx

```diff
@@ -12,7 +12,7 @@ type State = {
   alerts: Array<Alert>;
 };
 
-class Alerts extends React.Component<Props, State> {
+class SystemAlerts extends React.Component<Props, State> {
```

Comment:
This should probably live in `app/components` as it's similar in usage to `<Indicators>`, but not going to do that here.